File: app/src/main/java/com/sd/dialogview/handler/AppDialogConfirmViewHandler.java
package com.sd.dialogview.handler;

import com.sd.lib.dialogview.DialogConfirmView;
import com.sd.lib.dialogview.core.handler.IDialogConfirmViewHandler;

public class AppDialogConfirmViewHandler implements IDialogConfirmViewHandler {
    @Override
    public int getContentViewResId(DialogConfirmView dialogView) {
        return 0;
    }

    @Override
    public void onContentViewChanged(DialogConfirmView dialogView) {

    }

    @Override
    public void onAttachedToWindow(DialogConfirmView dialogView) {
        dialogView.setTextContent("hook content");
        dialogView.setTextContentSub("hook content sub");
    }

    @Override
    public void onDetachedFromWindow(DialogConfirmView dialogView) {

    }
}

File: app/src/main/java/com/sd/demo/dialogview/handler/AppDialogConfirmViewHandler.kt
package com.sd.demo.dialogview.handler

import com.sd.lib.dialogview.DialogConfirmView
import com.sd.lib.dialogview.core.handler.IDialogConfirmViewHandler

class AppDialogConfirmViewHandler : IDialogConfirmViewHandler {
    override fun getContentViewResId(dialogView: DialogConfirmView): Int {
        return 0
    }

    override fun onContentViewChanged(dialogView: DialogConfirmView) {

    }

    override fun onAttachedToWindow(dialogView: DialogConfirmView) {
        dialogView.setTextContent("hook content")
        dialogView.setTextContentSub("hook content sub")
    }

    override fun onDetachedFromWindow(dialogView: DialogConfirmView) {

    }
}